.github/workflows/ci.yml

```diff
@@ -37,8 +37,8 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
-          python-version: '3.11'
-          cache: 'pip'
+          python-version: 3.x
```

Comment:
TIL about x-ranges.

https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#using-the-python-version-input